### PR TITLE
feat: add landing page's intro

### PR DESCRIPTION
**Pull request by linooohon**
## Types of Changes

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description

- Add the intro section on the landing page.
- Remove  `NTU <div>` DOM on the landing page.
- The new file `components/intro` is for adding the intro section on the landing page, also the file `pages/index.vue` and the file `static` had some changed.

## Steps to Test This Pull Request

- Check on the landing page.

## ~~Issue(Solved)~~

- ~~Due to the Covid-19 situation, the description of the intro section needs to be updated,still waiting for the updating paragraph~~.

### CODE REVIEW

**Comment by linooohon** [comment]
Fixed the typing error and the length problem in 18n.js from [6372839](https://github.com/pycontw/pycontw-2021/commit/637283970962035c5dc48e542c83fa8e145dd3ea) to [52e6c61](https://github.com/pycontw/pycontw-2021/commit/52e6c61a49af85d743e50820c90187aede30d4d7)

File: components/intro/IntroMilestone.vue [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<template>
+    <div>
+        <p class="achievement">{{ $t(achievement) }}</p>
+        <p class="number">{{ $t(achievementNumber) }}</p>
+        <p class="achievement text-2xl mb-2">
+            {{ $t(achievementUnit) }}
```
Comment (mattwang44) [inline components/intro/IntroMilestone.vue:6]: The content passed in here has already translated.
```suggestion
        <p class="achievement">{{ achievement }}</p>
        <p class="number">{{ achievementNumber }}</p>
        <p class="achievement text-2xl mb-2">
            {{ achievementUnit }}
```

File: components/intro/IntroMilestone.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<template>
+    <div>
+        <p class="achievement">{{ $t(achievement) }}</p>
+        <p class="number">{{ $t(achievementNumber) }}</p>
+        <p class="achievement text-2xl mb-2">
```
Comment (mattwang44) [inline components/intro/IntroMilestone.vue:5]: nit: BEM (checkout [this post](https://css-tricks.com/bem-101/)) is recommended here. For example,
```suggestion
        <p class="milestone__achievement">{{ $t(achievement) }}</p>
        <p class="milestone__number">{{ $t(achievementNumber) }}</p>
        <p class="milestone__unit text-2xl mb-2">
```

Comment (linooohon) [reply components/intro/IntroMilestone.vue]: Thanks for the suggestion, I fixed the class names and refactored components to make HTML which is in the file `intro.vue` look more structured.

Comment (linooohon) [reply components/intro/IntroMilestone.vue]: (6c0cc63)[https://github.com/pycontw/pycontw-2021/pull/60/commits/6c0cc637baf82e12d72d28f9f8bcdaf30a46c481]

File: components/intro/IntroMilestone.vue (hunk 1 above)
Comment (linooohon) [reply components/intro/IntroMilestone.vue]: Okay, I see, so the contents will be translated in the `Intro.vue` component first, is that correct?

Comment (mattwang44) [reply components/intro/IntroMilestone.vue]: yes, which is already done by you actually.

**Comment by mattwang44** [comment]
add a commit 6744bcd for fixing the new warning after a later version of TailwindCSS is adopted in #65.

**Review by mattwang44 — APPROVED** [review]
LGTM

**Pull request merged**